project/CelebornBuild.scala

```diff
@@ -540,7 +576,20 @@ trait SparkClientProjects {
         ) ++ commonUnitTestDependencies
       )
   }
-  
+
+  def sparkColumnarShuffle: Project = {
```

Comment:
let's add the project `sparkColumnarShuffle` to the spark tests group `sparkGroup` to enable the columnar-related tests for SBT.

Reply:
Created a new PR #1854